README.md

```diff
@@ -17,19 +17,19 @@ Table of Contents
     * [License](#license)
 
 ## Usage
-Add it as an npm dev-dependency
+Add it as an npm devDependency, as well as all the necessary [peerDependencies](https://nodejs.org/en/blog/npm/peer-dependencies/).
 
 ```bash
-$ npm i eslint-config-commercetools --save-dev
+$ npm install --save-dev eslint-config-commercetools eslint@^#.#.# eslint-config-airbnb@^#.#.# eslint-plugin-jsx-a11y@^#.#.# eslint-plugin-import@^#.#.# eslint-plugin-react@^#.#.#
```

Comment:
> This script is just a help for whoever wants to add this config to its project to install those deps, but in the project itself, not this package.

@emmenko I am not sure it's just a help. It's required for the library to run. The way most of our libraries are setup now is that they have the dependencies in their package.json and what we were trying to achieve is that installing only this module install other require module for linting. So they need to add it to their CONTRIBUTING.md and we need to add the instructions that they should add it, so that linting will work for anybody using it.

Except I miss something.